pkg/vegeta/report.go

```diff
@@ -86,5 +86,22 @@ decode:
 		return json.Marshal(jsonReportResponse)
 	}
 
+	if format == TextFormat {
```

Comment:
```suggestion
	else if format == TextFormat {
```
Using `else if` will be more appropriate.